webapp/channels/src/reducers/views/admin.ts

```diff
@@ -67,6 +67,7 @@ export const adminConsoleUserManagementTablePropertiesInitialState: AdminConsole
     cursorUserId: '',
     cursorColumnValue: '',
     columnVisibility: {},
+    filterStatus: '',
```

Comment:
We don't need a default value for this. If there's nothing that we want to filter on, should this not be `undefined`?